### PR TITLE
add 'eth_getBalance'.

### PR DESCRIPTION
**Pull request by EighteenZi**
add jsonrpc interface: eth_getBalance.

### CODE REVIEW

**Review by driftluo — APPROVED** [review]

File: libproto/src/protos/request.proto [hunk 1]
```diff
@@ -39,6 +39,8 @@ message Request {
         bytes transaction_proof = 21;
         // cita_getMetaData
         string meta_data = 22;
+        // eth_getBalance
+        string balance = 23;
```
Comment (chaoticlonghair) [inline libproto/src/protos/request.proto:43]: I think the variable name is weired.
The content for this variable is not `balance`.

Comment (EighteenZi) [reply libproto/src/protos/request.proto:43]: The name 'balance' used here is more like an 'identifier', indicting its' usage, 
and the content includes all input parameters,  not the quantity of balance.
Just like code and abi above.

File: libproto/src/protos/response.proto [hunk 2]
```diff
@@ -34,6 +34,8 @@ message Response {
         bytes transaction_proof = 20;
         // cita_getMetaData
         string meta_data = 21;
+        // eth_getBalance
+        bytes balance = 22;
```
Comment (u2) [inline libproto/src/protos/response.proto:38]: The balance should be uint64 type.

**Review by TheWaWaR — APPROVED** [review]
ok

File: libproto/src/protos/response.proto (hunk 2 above)
Comment (TheWaWaR) [reply libproto/src/protos/response.proto:38]: Now `bytes` is for `U256` .

If we can ensure `uint64` is enough for total balance with smallest unit, then we should use `uint64`, it's more clearly.

Can we ensure this?

Comment (TheWaWaR) [reply libproto/src/protos/response.proto:38]: @EighteenZi Can we do some math for the `balance` thing? To prove uint64 maybe not enough.

Comment (EighteenZi) [reply libproto/src/protos/response.proto:38]: yes, i used u64 first.
But the type of 'balance' in 'Account ' is U256.
If it is casted into u64, there may be something unsafe.
2 ^ 64 = 18446744073709551616 = 1.8446744073709551616e+19
1 Ether = 10^{18} Wei
So in Ethereum, it MUST be U256.

In our applyment, I can't decide which to select, u64 or u256.

Needs more discussion and dicision.

**Review by chaoticlonghair — APPROVED** [review]

File: libproto/src/protos/response.proto (hunk 2 above)
Comment (u2) [reply libproto/src/protos/response.proto:38]: ok, as you say.

No problem.

**Pull request merged**